xxh3.h

```diff
@@ -1336,6 +1336,48 @@ XXH3_scrambleAcc(void* XXH_RESTRICT acc, const void* XXH_RESTRICT secret)
 #  define XXH_PREFETCH_DIST_AVX512_128 512
 #endif
 
+/* The prefetch feature will be affected my many factors. For internal loop, we control it by three
+ * macros:
+ *   - XXH_PREFETCH
+ *   - XXH_PREFETCH_DIST_64
+ *   - XXH_PREFETCH_DIST_128
+ *
+ * Given XXH_PREFETCH is enabled, here we can select proper prefetch distances at compile time over
+ * (compiler, vector modes, target hardwares). Each group distances should be selected over
+ * profiling experiments. And we still have some principles:
+ *   - the base line comes from disabling prefetch
+ *   - since the internal loop step is forward, the distances should be non-negative
+ *   - distances shoule be aligned to the common cache line size of the target hardwares
+ *   - lean towards smaller distances, and basically stop early, as soon as benefits from larger
+ *     prefetching distances start to become "small". That's because it preserves a general resource
+ *     (memory bandwidth) for the benefit of the larger system, be it other parts of the application
+ *     or other concurrent applications.
+ */
+#if defined(_MSC_VER) // MSVC
+// use common settings
+#  define XXH_PREFETCH_DIST_64  384
+#  define XXH_PREFETCH_DIST_128 384
+#elif defined(__clang__) // clang
+#  if (XXH_VECTOR == XXH_AVX512) // avx implies x86
+#    define XXH_PREFETCH_DIST_64  320
+#    define XXH_PREFETCH_DIST_128 320
+#  else
+#    define XXH_PREFETCH_DIST_64  384
+#    define XXH_PREFETCH_DIST_128 384
+#  endif
+#elif defined(__GNUC__) && ! defined(__clang__) // GCC
+#  if (XXH_VECTOR == XXH_AVX512) // avx implies x86
+#    define XXH_PREFETCH_DIST_64  640
```

Comment:
Is it still valuable to separate `XXH_PREFETCH_DIST_64` from `XXH_PREFETCH_DIST_128` if the only difference is here ?
It seems this complexifies the code, for little gains.